neat_addr.h

```diff
@@ -38,6 +39,8 @@ struct neat_addr {
     uint8_t __pad;
     uint16_t __pad2;
     uint8_t prefix_length;
+
+    struct pvds pvds;
```

Comment:
Ok, I wasn't sure you meant that.

So I did what you propose in bbe3202, everything is stored in `neat_pvd->results`. And when we want to access the PvDs of an address, we simply iterate on the results until we find the one corresponding.